docs/content/users/providers/pantheon.md

```diff
@@ -36,23 +36,52 @@ If you have DDEV installed, and have an active Pantheon account with an active s
 
 5. Configure the local checkout for DDEV using [`ddev config`](../usage/commands.md#config).
 
-6. Add `PANTHEON_SITE` and `PANTHEON_ENVIRONMENT` variables to your project `.ddev/config.yaml`:
+6. Add `DDEV_PANTHEON_SITE` and `DDEV_PANTHEON_ENVIRONMENT` variables to your project `.ddev/config.yaml`:
 
     ```yaml
     web_environment:
-        - PANTHEON_SITE=yourprojectname
-        - PANTHEON_ENVIRONMENT=dev
+        - DDEV_PANTHEON_SITE=yourprojectname
+        - DDEV_PANTHEON_ENVIRONMENT=dev
     ```
 
-    You can also do this with `ddev config --web-environment-add="PANTHEON_SITE=yourprojectname,PANTHEON_ENVIRONMENT=dev"`.
+    You can also do this with `ddev config --web-environment-add="DDEV_PANTHEON_SITE=yourprojectname,DDEV_PANTHEON_ENVIRONMENT=dev"`.
 
     You can usually use the site name, but in some environments you may need the site ID, which is the long third component of your site dashboard URL. So if the site dashboard is at `https://dashboard.pantheon.io/sites/009a2cda-2c22-4eee-8f9d-96f017321555#dev/`, the site ID is `009a2cda-2c22-4eee-8f9d-96f017321555`.
 
     Instead of setting the environment variables in configuration files, you can use
-    `ddev pull pantheon --environment=PANTHEON_SITE=yourprojectname,PANTHEON_ENVIRONMENT=dev` for example.
+    `ddev pull pantheon --environment=DDEV_PANTHEON_SITE=yourprojectname,DDEV_PANTHEON_ENVIRONMENT=dev` for example.
+
+    !!!note "Legacy Variable Names"
+        The old `PANTHEON_SITE` and `PANTHEON_ENVIRONMENT` variable names are still supported for backward compatibility but are deprecated. These names conflict with Pantheon's own environment variables, which can cause applications to misidentify their runtime environment. Using `DDEV_PANTHEON_SITE` and `DDEV_PANTHEON_ENVIRONMENT` is strongly recommended.
 
 7. Run [`ddev restart`](../usage/commands.md#restart).
 
 8. Run `ddev pull pantheon`. DDEV will download the Pantheon database and files and bring them into the local DDEV environment. You should now be able to access the project locally.
 
 9. Optionally use `ddev push pantheon` to push local files and database to Pantheon. The [`ddev push`](../usage/commands.md#push) command can potentially damage your production site, so we don't recommend using it.
+
+## Using Existing Backups vs Fresh Database Dumps
+
+By default, `ddev pull pantheon` generates a fresh database dump, which ensures you get the most current data but can take several minutes. For faster pulls, you can use existing Pantheon backups instead.
+
+To use existing backups, set `PANTHEON_USE_BACKUP=true`:
```

Comment:
Would it make sense to use `DDEV_PANTHEON_USE_BACKUP` ?

Just thinking about a situation where people might confuse it with a variable coming from Pantheon / or if Pantheon adds it.

Reply:
Agreed. Or DDEV_USE_PANTHEON_BACKUP